jOOL-java-8/src/main/java/org/jooq/lambda/Agg.java

```diff
@@ -941,8 +941,118 @@ else if (percentile == 1.0)
     }
 
     /**
-     * Get a {@link Collector} that calculates the common prefix of a set of strings.
+     * CS304 Issue link: https://github.com/jOOQ/jOOL/issues/221
```

Comment:
May I ask what CS304 means? I'll remove it again, but I'm curious. Is this some CS=Computer Science university class?

Reply:
I think we should apologize for these comments. We add these PRs because it is our software engineering course homework to make contributions on some open project╮(╯﹏╰）╭

Reply:
No worries, just curious. I'll remove the comments. Another question, what made you choose jOOλ?

Reply:
It's mainly because this project is easy to learn, and it seems to be an important project because it's popular. Actually we have also considered JOOQ, but it's too big.

Reply:
Makes sense. Yes, it's much harder to make meaningful contributions to jOOQ - especially because the tests are not open source.